.github/workflows/main.yml

```diff
@@ -0,0 +1,115 @@
+name: 🚀 CI/CD for Overflow
+
+on:
+  push:
+    branches: [master]
+
+  pull_request:
+    branches: [master]
+
+env:
+  node_version: 10.14
```

Comment:
@kirill-panoply please change it to node 14

Reply:
I don't think we should do it since we're using this package in applications which are running on node 10.14